app/jobs/cache_analytics_job.rb

```diff
@@ -5,38 +5,27 @@ class CacheAnalyticsJob < ApplicationJob
   queue_as :default
 
   def perform(*_args)
-    Analytic.transaction do
-      analytics_by_jur_path = {}
-      root_jurs = []
+    Jurisdiction.all.find_each do |jur|
```

Comment:
I'm not sure you need to call `all` as well as `find_each` from my testing it looks like they produce equal queries
* `[ActiveJob] [CacheAnalyticsJob] [003bc49f-8352-4671-b649-ffe2868b8898]   Jurisdiction Load (0.4ms) SELECT `jurisdictions`.* FROM `jurisdictions` ORDER BY `jurisdictions`.`id` ASC LIMIT 1000`
* `[ActiveJob] [CacheAnalyticsJob] [6f7d4aba-c387-4971-951e-9cdef90237ae]   Jurisdiction Load (0.4ms) SELECT `jurisdictions`.* FROM `jurisdictions` ORDER BY `jurisdictions`.`id` ASC LIMIT 1000`